packages/bridge/src/runtime/nitro/renderer.ts

```diff
@@ -234,8 +234,8 @@ export default defineRenderHandler(async (event) => {
         HEAD_ATTRS: joinAttrs(htmlContext.headAttrs),
         HEAD: joinTags(htmlContext.head),
         BODY_ATTRS: joinAttrs(htmlContext.bodyAttrs),
-        BODY_PREPEND: joinTags(htmlContext.bodyPreprend),
-        APP: _rendered.html + joinTags(htmlContext.bodyAppend)
+        BODY_PREPEND: joinTags(htmlContext.bodyPrepend),
+        APP: joinTags(htmlContext.bodyPrepend) + _rendered.html + joinTags(htmlContext.bodyAppend)
```

Comment:
Is there a reason that `bodyPrepend` is included twice?

Reply:
You are right, sorry. The `BODY_PREPEND` placeholder doesn't exist in the template so I removed it (https://github.com/nuxt/bridge/pull/837/commits/9fd7ad6c08a14c9115faa781d6d0ff2921e5fae7).